src/store/actions/movies.js

```diff
@@ -0,0 +1,60 @@
+export const sortByTitle = (movies) => {
+  const sortedMoviesData = movies.sort(function (a, b) {
+    //first argument should appear before second argument
+    if (a.title.toLowerCase() < b.title.toLowerCase()) return -1;
+    //first argument should appear after second argument
+    if (a.title.toLowerCase() > b.title.toLowerCase()) return 1;
+    // if equal return zero
+    return 0;
+  });
+
+  return {
+    type: "SORT_BY_TITLE",
+    payload: {
+      sortedMovies: sortedMoviesData,
```

Comment:
Same as for "src/store/thunks/movieDetails" - redundant remapping of the payload.

Reply:
redundant remapping of payload removed and resolved